website_snippet_carousel_product/static/src/scss/s_product_carousel.scss

```
@@ -0,0 +1,31 @@
+// Copyright 2020 Tecnativa - Alexandre Díaz
+// License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+.s_product_carousel {
+    .oe_product {
+        border-width: 0;
+
+        .oe_product_image {
+            position: unset;
+            padding: 15px;
```

I wonder if you can replace this for something similar to:

```suggestion
            padding: $grid-gutter-width / 2;
```

Or otherwise use rem units as the rest of BS4 please.

---

Refactored